src/main/java/core/basesyntax/services/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.services;
+
+import core.basesyntax.figures.*;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure(){
+        ColorSupplier service = new ColorSupplier();
+        String color = service.getRandomColor();
```

Comment:
We already have ```Color``` enum, why do we need color to be ```String```?

Reply:
Thank you! Have changed this.